Source/DSPSpy/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.5.0)
+
+project(dspspy)
+
+set(CMAKE_EXECUTABLE_SUFFIX ".elf")
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+
+set(MACHDEP "-DHW_RVL -DRVL -DGEKKO -mrvl -mcpu=750 -meabi -mhard-float")
+set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS} ${MACHDEP}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${MACHDEP}")
+
+include_directories(${LIBOGCDIR}/include)
```

Comment:
should this not use `target_include_directories`?

Reply:
You're correct. Just to be sure, the target name would be the project name, `dspspy`?

Reply:
It would be `main`. `add_executable` and `add_library` introduce build targets. So they would need to go below the `add_executable` call in this case as well.